features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/components/APIDetailsTopMenu.jsx

```diff
@@ -84,21 +85,21 @@ const APIDetailsTopMenu = (props) => {
                     State
                 </Typography>
             </div>
-
-            <VerticalDivider height={70} />
-
-            <a
-                target='_blank'
-                rel='noopener noreferrer'
-                href={`${window.location.origin}/store-new/apis/${api.id}/overview`}
-                className={classes.viewInStoreLauncher}
-            >
-                <div>
-                    <LaunchIcon />
-                </div>
-                <div className={classes.linkText}>View In store</div>
-            </a>
             <VerticalDivider height={70} />
+            {isVisibleInStore &&
+                <a
+                    target='_blank'
+                    rel='noopener noreferrer'
+                    href={`${window.location.origin}/store-new/apis/${api.id}/overview`}
```

Comment:
Get the `store-new` context from configurations

Reply:
Like we have done here

https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/Publisher.jsx#L164

Reply:
Fixed :)